src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -272,14 +274,14 @@ private void updateUsedStateForColumns(IReadOnlyList<DataColumnDTO> dataColumnDT
          dataColumnDTOs.Each(dto => dto.Used = used);
       }
 
-      private void raiseUsedChanged(IReadOnlyList<DataColumn> dataColumns, bool used)
+      private void raiseUsedChanged(IReadOnlyList<DataColumn> dataColumns, bool used, bool isLinkedDataToSimulations = false)
```

Comment:
I don't understand why this parameter, wherevever it's defined, is always optional. in private method, it should not be. Entry point fine, but after this, this is not easier to read
I Woudl argue that this should never be optional but I am not sure